gradle.properties

```diff
@@ -1,2 +1,2 @@
-checkstyleMainMaxWarnings=4926
+checkstyleMainMaxWarnings=4959
```

Comment:
Warning count probably only needs to be bumped by two.  You can verify by running `./gradlew clean checkstyleMain` and looking at the count in the report located at `build/reports/checkstyle/main.html`.